clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -10970,8 +10970,10 @@ def warn_ivdep_redundant : Warning <"ignoring redundant Intel FPGA loop "
   "attribute 'ivdep': safelen %select{INF|%1}0 >= safelen %select{INF|%3}2">,
   InGroup<IgnoredAttributes>;
 def warn_attribute_spelling_deprecated : Warning<
-  "attribute %0 is deprecated, did you mean to use 'reqd_sub_group_size' instead?">,
+  "attribute %0 is deprecated">,
   InGroup<DeprecatedAttributes>;
+def note_spelling_suggestion : Note<
+  "did you mean to use 'intel::reqd_sub_group_size' instead?">;
```

Comment:
Make intel::reqd_sub_group_size a %0 instead, then fill it in with your Diag call.

Reply:
I was actually surprised a general 'did you mean' like this one did not exist already.

Reply:
@elizabethandrews : the only one I found was note_function_suggestion, which is textually identical to what I'd like, but is function-specific.  I'm tempted to update the name in the upstream to make it work for anything...

Reply:
> Make intel::reqd_sub_group_size a %0 instead, then fill it in with your Diag call.

Done.